#! /bin/sh /usr/share/dpatch/dpatch-run
## DP:  fix for output of debug information:
##      http://www.zabbix.com/forum/showpost.php?p=23046&postcount=1

@DPATCH@
--- zabbix-1.4.1/src/libs/zbxsysinfo/common/common.c	(revision 4406)
+++ zabbix-1.4.1/src/libs/zbxsysinfo/common/common.c	(working copy)
@@ -232,7 +232,7 @@
 		
 
 #else /* not _WINDOWS */
-	command = zbx_dsprintf(command, param);
+	command = zbx_dsprintf(command, "%s", param);
 
 	if(0 == (hRead = popen(command,"r")))
 	{
